src/test/java/io/wisetime/connector/jira/JiraConnectorPostTimeTest.java

```diff
@@ -312,9 +305,127 @@ void postTime_should_handle_tags_not_matching_project_keys_filter() {
     verifyJiraNotUpdated();
   }
 
+  @Test
+  void postTime_check_narrative_duration_divide_between_tags() {
+    final List<Tag> tags = ImmutableList.of(
+        fakeEntities.randomTag("/Jira/"), fakeEntities.randomTag("/Jira/")
+    );
+
+    final TimeRow timeRow1 = fakeEntities.randomTimeRow().activityHour(2018110110).durationSecs(2400);
+    final TimeRow timeRow2 = fakeEntities.randomTimeRow().activityHour(2018110109).durationSecs(66);
```

Comment:
The test doesn't assert against the times, so they could be removed from the fake entries - TimeRows `activityHour(2018110109).durationSecs(66)` and also from the TimeGroup - `.totalDurationSecs(3000).durationSplitStrategy(TimeGroup.DurationSplitStrategyEnum.DIVIDE_BETWEEN_TAGS)`. In the next 2 tests as well.

Reply:
Sorry, didn't notice your comment. I have updated the PR now. Thanks